addons/medical/functions/fnc_canTreatSAMSplint.sqf

```diff
@@ -3,8 +3,9 @@
  * Check if can treat Splint
  *
  * Arguments:
- * 0: The patient <OBJECT>
- * 1: SelectionName <STRING>
+ * 1: The caller (not used, can pass nil) <OBJECT>
```

Comment:
I'd say remove the `(not used, can pass nil)` part.
Also this should be argument 0 not `1`
IMO it's good to have this here. Makes it easily extendable later. In case someone comes up with a requirement on the caller.

Reply:
Roger, will fix that. Would it be better to skip the caller with "" or just accept to _caller for later extension?

Reply:
doesn't really make a difference I think.